deepspeed/runtime/engine.py

```diff
@@ -924,12 +924,6 @@ def _do_args_sanity_check(self, args):
                     args, "deepspeed_config") and args.deepspeed_config is not None
             ), "DeepSpeed requires --deepspeed_config to specify configuration file"
 
-            assert os.path.isfile(
-                args.deepspeed_config
```

Comment:
Rather than removing this assert, could we instead do correctness check for base64 encoded config? It is better for failures to occur early here during sanity check than later in DeepSpeedConfig constructor.

Reply:
the autotuner no longer needs this encoding feature, closing the PR. thanks for the comment